sdk/managed/test/Microsoft.WindowsAzure.MobileServices.Test/FunctionalTests/FromSdkTests/MobileServiceTableTest.cs

```diff
@@ -578,124 +578,124 @@ public class MobileServiceTableGenericFunctionalTests : FunctionalTestBase
             await stringIdTable.DeleteAsync(item);
         }
 
-        [AsyncTestMethod]
-        [Tag("NodeRuntimeOnly")]
-        public async Task AsyncTableOperationsWithStringIdAgainstIntegerIdTable()

```

Comment:
This test is node only, tests really specific error messages and is largely irrelevant - too much effort to fix. We'll consider testing this scenario when we refactor.